website/content/en/docs/concepts/services.md

```diff
@@ -1,26 +1,204 @@
 
-### Importing and binding a Service
+The Import instance creates the service endpoint in the same namespace as it is
+ defined in. The created service will have the Import's `metadata.Name`. This
+ allows maintaining independent names for services between peers[^TLS]. Alternately,
+ you may use the same name for the import and related source exports.
+ You can define multiple Import CRDs for the same set of Exports in different
+ namespaces. These are independent of each other.
+
+{{% expand summary="Import Custom Resource" %}}
+
+```go
+type Import struct {
+    metav1.TypeMeta   `json:",inline"`
+    metav1.ObjectMeta `json:"metadata,omitempty"`
+    
+    Spec ImportSpec     `json:"spec"`
+    Status ImportStatus `json:"status,omitempty"`
+}
+
+type ImportSpec struct {
+    Port uint16 `json:"port"`
+    TargetPort uint16 `json:"targetPort,omitempty"`
+    Sources []ImportSource `json:"sources"`
+    LBScheme string `json:"lbScheme"`
+}
+
+type ImportSource struct {
+    Peer string `json:"peer"`
+    ExportName string `json:"exportName"`
+    ExportNamespace string `json:"exportNamespace"`
+}
+
+type ImportStatus struct {
+    Conditions []metav1.Condition `json:"conditions,omitempty"`
+}
+```
+
+{{% /expand %}}
+
+The ImportSpec defines the following fields:
+
+- **Port** (integer, required): the imported, user facing, port number define
+ on the created service object.
+- **TargetPort** (integer, optional): this is the internal listening port
+ used by the data plane pods to represent the remote services. Typically the
+ choice of TargetPort should be left to the ClusterLink control plane, allowing
+ it to select a random and non-conflicting port, but there may be cases where
+ you wish to assume responsibility for port selection (e.g., a-priori define
+ local cluster Kubernetes NetworkPolicy object instances). This may result in
+ [port conflicts](https://kubernetes.io/docs/concepts/services-networking/service/#avoid-nodeport-collisions)
+ as is done for NodePort services.
+- **Sources** (source array, required): references to remote exports providing backends
+ for the Import. Each reference names a different export through the combination of
+  - *Peer* (string, required): name of ClusterLink peer where the export is defined.
+  - *ExportNamespace* (string, required): name of the namespace on the remote peer where
+   the export is defined.
+  - *ExportName* (string, required): name of the remote export.
+- **LBScheme** (string, optional): load balancing method to select between different
+ Sources defined. The default policy is `random`, but you could override it to use
+ `round-robin` or `static` (fixed) assignment.
+
+<!-- Importing multiport? It is not possible... Could use merge in future?
+ perhaps, but might requires explicit service name so can merge correctly
+ or use port set instead of individual port per export/import -->
+
+As with exports, importing a Service does not automatically make it accessible by
```

Comment:
Maybe need to add that the connection is not open yet, just when traffic arrives.

Reply:
it should be clear since we write about service and needing to add a policy to it